building error

Hello.
I am trying build panda as:
```
mkdir build; cd build
../build.sh i386-softmmu,x86_64-softmmu
```
And I've the following error:
```
    inlined from ‘vvfat_open’ at /opt/panda/block/vvfat.c:1181:9:
/opt/panda/block/vvfat.c:426:14: error: writing 1 byte into a region of size 0 [-Werror=stringop-overflow=]
  426 |     dest[2*i]=dest[2*i+1]=0;
      |     ~~~~~~~~~^~~~~~~~~~~~~~

```

Thank you!

--- a/build.sh
+++ b/build.sh
@@ -65,8 +65,9 @@ gcc --version | awk '/gcc/ && ($3+0)<7.1{print "Fatal error: GCC too old"; exit 
 g++ --version | awk '/g\+\+/ && ($3+0)<7.1{print "Fatal error: G++ too old"; exit 1}' || exit 1
 
 # Untested GCC - it's probably going to have some warnings - Just disable Werror and hope it works
-gcc --version | awk '/gcc/   && ($3+0)>9.3{print "WARNING: Your GCC is too new: disabling -Werror and hoping this builds"; exit 1}' || COMPILER_CONFIG="--extra-cflags=-Wno-error"
-g++ --version | awk '/g\+\+/ && ($3+0)>9.3{print "WARNING: Your G++ is too new: disabling -Werror and hoping this builds"; exit 1}' ||  COMPILER_CONFIG="--extra-cxxflags=-Wno-error"
+COMPILER_CONFIG=""
+gcc --version | awk '/gcc/   && ($3+0)>9.3{print "WARNING: Your GCC is too new: disabling -Werror and hoping this builds"; exit 1}' || COMPILER_CONFIG+= "--extra-cflags=-Wno-error"
+g++ --version | awk '/g\+\+/ && ($3+0)>9.3{print "WARNING: Your G++ is too new: disabling -Werror and hoping this builds"; exit 1}' ||  COMPILER_CONFIG +=" --extra-cxxflags=-Wno-error"
 
 ### Check for protobuf v2.
 if ! pkg-config --exists protobuf; then

